consumer.go

```diff
@@ -115,7 +126,7 @@ func (c *segmentConsumer) Close() {
 	// this will also make the loops in the reader goroutines stop
 	err := c.reader.Close()
 	if err != nil {
-		sdk.Logger(context.Background()).Err(err).Msg("couldn't close reader")
+		fmt.Printf("couldn't close reader: %v\n", err)
```

Comment:
Think might be a copy/paste gone wrong. Oh and, didn't we say we should return this error instead of logging it? Maybe now is the time 🙂